src/controllers/hotkeys/HotkeyController.hpp

```diff
@@ -114,6 +121,13 @@ class HotkeyController final : public Singleton
      **/
     static void showHotkeyError(const std::shared_ptr<Hotkey> &hotkey,
                                 QString warning);
+    /**
+     * @brief finds a Hotkey matching category, action and neededArguments.
+     * Accepted if and only if the category matches, the action matches and neededArguments match(TODO)
+     * */
```

Comment:
Is this still TODO?
If no, could you update with the logic for how `neededArguments` should be handled? i.e. if there's no direct match, should it look for non-direct matches?

Reply:
I forgot to update this, it should work. I use `neededArguments` a bunch, for example src/widgets/splits/SplitHeader.cpp:353, it requires hotkey arguments like {"split"}.